build.sbt

```diff
@@ -338,6 +341,13 @@ lazy val javalib =
 lazy val assembleScalaLibrary = taskKey[Unit](
   "Checks out scala standard library from submodules/scala and then applies overrides.")
 
+lazy val auxlib =
+  project
+    .in(file("auxlib"))
+    .settings(libSettings)
+    .settings(mavenPublishSettings)
+    .dependsOn(nativelib)
```

Comment:
I think that `auxlib` doesn't need to depend on `nativelib`, but it doesn't hurt to keep the dependency.

Reply:
It does have to depend on it, otherwise there is no `native.Ptr` when you compile it with a scala native plugin, and it crashes due to not being able to resolve the symbol for it.